main.py

```diff
@@ -1,29 +1,83 @@
 """
 This is an Add-On for DocumentCloud which wraps the tabula-py library
 https://github.com/chezou/tabula-py
-
-It attempts to convert tables from PDFs to CSVs
+It allows you to export tables from a set of PDFs into CSVs and
+download the resulting CSVs as a zip file.
 """
 
+import os
+import sys
+import zipfile
+import subprocess
+from urllib.parse import urlparse
+
+import requests
 import tabula
+
 from documentcloud.addon import AddOn
+from clouddl import *
```

Comment:
Avoid star imports, it makes it hard to tell where things are imported from - either do `import clouddl` and use qualified references or import each thing you are using individually